incubator/java-openliberty/image/Dockerfile-stack

```diff
@@ -34,7 +34,7 @@ ENV APPSODY_USER_RUN_AS_LOCAL=true
 # Set APPSODY_DEV_MODE to get in the habit in case we need it someday
 ENV APPSODY_PREP="export APPSODY_DEV_MODE=prep;  ../validate.sh"
 
-ENV APPSODY_RUN="export APPSODY_DEV_MODE=run; mvn -B -Pstack-image-run -DappsDirectory=apps -Dmaven.repo.local=/mvn/repository liberty:dev"
+ENV APPSODY_RUN="export APPSODY_DEV_MODE=run; /project/invoke-appsody.sh"
```

Comment:
You'd need to do this for test/debug also.

I have a better idea..let's just move this function into validate.sh, (where we'll continue to test APPSODY_DEV_MODE so we don't run it in `appsody build`), since that runs in PREP.